README.md

```diff
@@ -63,15 +63,15 @@ Pick can also easily be used from within Vim with this VimL function:
 ```viml
 function! PickCommand(choice_command, pick_args, vim_command)
   try
-    let selection = system(a:choice_command . " | ./src/pick" . a:pick_args)
+    let selection = system(a:choice_command . " | pick" . a:pick_args)
+    if v:shell_error == 0
+      exec a:vim_command . " " . selection
+    endif
   catch /Vim:Interrupt/
     " Swallow the ^C so that the redraw below happens; otherwise there will be
```

Comment:
Yes. I mistakenly removed this in an earlier commit since it had no effect on my Arch Linux machine. This made it fail on @teoljungberg's Mac. Instead of returning we fall through to the redraw on line 74.